202103_bicepkeck/bicepkeck.toml

```diff
@@ -0,0 +1,50 @@
+[experiment]
+
+total_experiment_length_years = 6
+
+
+# Number of splits, 1 generates only full mission
+# 2-7 generates the full mission map and then the requested number
+# of splits scaled accordingly. E.g. 7 generates the full mission
+# map and 7 equal (yearly) maps
+number_of_splits = 1
+
+include_atmosphere = false
+
+include_noise = false
+
+# between 0 and 1, typically 0.2
+# remove default so the code errors if a band is not specified
+# observing_efficiency.default = 0.2
+
+# we can override the observing efficiency for specific sites/bands
+# always specify first site, then telescope, then band
+
+# Copy-pasted from https://docs.google.com/spreadsheets/d/1jR9gSsJ0w1dEO5Jb_URlD3SWYtgFtwBgB3W88p6puo0/edit#gid=1776795091
+# on 4 March 2021
+
+observing_efficiency.Pole.SAT.LFS1  = 0.194656896
```

Comment:
These factors include the 20% detector yield penalty: https://docs.google.com/spreadsheets/d/1jR9gSsJ0w1dEO5Jb_URlD3SWYtgFtwBgB3W88p6puo0/edit?pli=1#gid=1776795091 We don't expect detector counts to scale the simulated atmosphere. Is the yield penalty taken out somewhere else or are we simply simulating 20% more atmosphere? Obviously not a huge issue, but best to know for certain.

Reply:
no, it's not taken out.
I should use here the the ones without yield and then have another parameter for yield?

Reply:
That would be ideal, yes. However, the issue only affects the atmospheric scaling where the overall calibration uncertainty is much larger than 20%. If you are modifying the design tool, I would also ask to correct for the thinning factors in atmospheric T->P leakage by dividing Q/U with `sqrt(thinfp)`.

Reply:
I'm always modifying the design tool, sure, can you open another issue with details on how to apply that fix?

Reply:
ok have 2 pull requests with these improvements:

* https://github.com/CMB-S4/s4_design_sim_tool/pull/23
* https://github.com/CMB-S4/s4_design_sim_tool/pull/24